projects/cdk/directives/platform/platform.directive.ts

```diff
@@ -1,20 +1,12 @@
-import {Directive, HostBinding, Input} from '@angular/core';
-import {TUI_PLATFORM, tuiPlatformFactory} from '@taiga-ui/cdk/tokens';
+import {Directive, HostBinding, inject, Input} from '@angular/core';
+import {TUI_PLATFORM} from '@taiga-ui/cdk/tokens';
 import {TuiPlatform} from '@taiga-ui/cdk/types';
 
 @Directive({
     selector: '[tuiPlatform]',
-    providers: [
-        {
-            provide: TUI_PLATFORM,
-            deps: [TuiPlatformDirective],
-            useFactory: ({tuiPlatform}: TuiPlatformDirective) =>
-                tuiPlatform || tuiPlatformFactory(),
```

Comment:
Don't remove this for now, just change `tuiPlatformFactory()` to `inject(TUI_PLATFORM, { skipSelf: true })`

Reply:
fixed